python/ray/autoscaler/_private/gcp/node_provider.py

```diff
@@ -164,14 +164,17 @@ def internal_ip(self, node_id: str):
             return ip
 
     @_retry
-    def create_node(self, base_config: dict, tags: dict, count: int) -> None:
+    def create_node(
+        self, base_config: dict, tags: dict, count: int
+    ) -> List[Tuple[dict, str]]:
```

Comment:
Thanks for the PR!

Could you add a doc string explaining the meaning of the returned data?
Also, could you modify the return type on the create node method in the NodeProvider interface to `Any` and change the docstring there to reflect the more general return type?
https://github.com/ray-project/ray/blob/905258dbc19753c81039f993477e7ab027960729/python/ray/autoscaler/node_provider.py#L121-L124

Reply:
Sounds good! I made some changes to the return value to make it consistent with NodeProvider and AWSNodeProvider, as well as added a doc string. Let me know if it looks better for you.